addons/pronelauncher/config.cpp

```diff
@@ -0,0 +1,25 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {
+            "ace_medical",
```

Comment:
Why is ace_medical a dependency? Also, ace_common already has ace_main as dependency

Reply:
https://github.com/acemod/ACE3/blob/cd2619a4d14ef5aef70b3e2726d5a43f733800ed/addons/pronelauncher/config.cpp#L9